### PR TITLE
Implement basic mixed paradigm

### LINKED ISSUE #277: Basic mixed paradigm
Should show 3 > 1,2 rather than 3', 3 would be much more common and thus the logical choice

### PR DESCRIPTION
**Pull request by eddieantonio**
I ran `update-layouts` to— update the layouts! — based on contributions from @aarppe. I also added a test for this.

Fixes #277

### CODE REVIEW

**Comment by aarppe** [comment]
So, it seems that from a bird's eye perspective things are fine, as coverage does not change (no code should change) and the tests pass. Presumably I needn't look at the code, as that should not have changed. I can also find the new test specification created for the TA paradigm, but where do I find the video of the test, as that is perhaps the one thing I'd check as to the change working as it should (before approving)?

File: cypress/integration/regressions.spec.js [hunk 1]
```diff
@@ -171,4 +171,17 @@ context('Regressions', () => {
...
       })
   })
+
+  it('should show 3>1,2 rather than 3\', 3 in the VTA layout', function () {
+    // Go to a VTA word:
+    cy.visit('/search/wâpamêw')
+    cy.contains('a', 'wâpamêw')
+      .click()
+
+    // N.B.: This test will fail if the word changes. ¯\_(ツ)_/¯
```
Comment (Madoshakalaka) [inline cypress/integration/regressions.spec.js]: ¯\_(ツ)_/¯

Comment (aarppe) [reply cypress/integration/regressions.spec.js]: I can see the test file here, but where's the video clip?

Comment (eddieantonio) [reply cypress/integration/regressions.spec.js]: That's a great question! Typically cypress comments and adds the video clip, but now you have to dig through the dashboard and find it. Here it is: https://dashboard.cypress.io/projects/8r2xra/runs/369/specs

Comment (eddieantonio) [reply cypress/integration/regressions.spec.js]: Honestly, the video clip is too fast to really notice anything :/

**Pull request merged**